crypto/asn1/a_mbstr.c

```diff
@@ -272,12 +270,15 @@ static int out_utf8(unsigned long value, void *arg)
 static int type_str(unsigned long value, void *arg)
 {
     unsigned long types;
+    const int native = ossl_fromascii(value);
```

Comment:
Implementation-defined, sorry. But on certain level one can probably say it's appropriate to avoid implementation-defined just as well as undefined.